Makefile

```diff
@@ -1,60 +1,23 @@
-GTEST_DIR ?= external/googletest/googletest
-CXX ?= clang++
-CXXFLAGS += -std=c++14 -pedantic -Wall -Wextra -Iinclude/
-CC ?= clang
-CFLAGS += -std=c99 -pedantic -Wall -Wextra -Wno-unused-parameter -Iinclude/
-AR ?= ar
-RANLIB ?= ranlib
-XCODEBUILD ?= xcodebuild
+build := build
+cmake := cmake
+make := make
 
-SOURCES = $(shell find src -name '*.cpp')
-OBJECTS = $(SOURCES:.cpp=.o)
-LIBRARY = libArbiter.a
+all:
+	mkdir -p $(build)
+	$(cmake) -B$(build) -H. -G "Unix Makefiles" -DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=TRUE
```

Comment:
Does `cmake` have more verbose (spelled out) variants of these flags? If so, it'd be nice to use them for understandability.